submissions/denismaysov/html-css-popup/index.html

```diff
@@ -0,0 +1,295 @@
+                          height="35"
+                          alt=""
+                          class="header__popup__img"
+                        />
+                        Chat
+                      </a>
+                    </li>
+                    <li class="header__popup_item">
+                      <a href="#" class="header__popup_link link">
+                        <img
+                          src="assets/blogs.png"
+                          width="48"
+                          height="35"
+                          alt=""
+                          class="header__popup__img"
+                        />
+                        Blogs
+                      </a>
+                    </li>
+                    <li class="header__popup_item">
+                      <a href="#" class="header__popup_link link">
+                        <img
+                          src="assets/mars.png"
+                          width="48"
+                          height="35"
+                          alt=""
+                          class="header__popup__img"
+                        />
+                        Mars
+                      </a>
+                    </li>
+                    <li class="header__popup_item">
+                      <a href="#" class="header__popup_link link">
+                        <img
+                          src="assets/mic.png"
+                          width="48"
+                          height="35"
+                          alt=""
+                          class="header__popup__img"
+                        />
+                        Microphone
+                      </a>
+                    </li>
+                    <li class="header__popup_item">
+                      <a href="#" class="header__popup_link link">
+                        <img
+                          src="assets/setting.webp"
+                          width="48"
+                          height="35"
+                          alt=""
+                          class="header__popup__img"
+                        />
+                        Settings
+                      </a>
+                    </li>
+                  </ul>
+                </div>
+              </div>
+            </li>
+            <li class="header__list_item">
+              <a href="#" class="header__link focus_item">
+                <img src="assets/bell.png" width="25" height="25" aria-label="Receive notifications" />
+              </a>
+            </li>
+            <li class="header__list_item">
+              <a
+                href="https://github.com/denismaysov"
+                target="_blank"
+                class="header__link focus_item"
+              >
+                <img
+                  src="assets/favicon.png"
+                  class="header__photo focus_item"
+                  width="25"
+                  height="25"
+                  aria-label="MyGitHub"
+                />
+              </a>
```

Comment:
The `aria-label` attribute better move to the link instead of image. Also, add an `alt` attribute with an empty value, because `alt` is a required attribute.